gnomic/__init__.py

```diff
@@ -1,279 +1 @@
-        """
-
-        :param str string: The genotype definition to parse
-        :param Genotype parent:
-        :param organisms: A list or tuple of :class:`Organism` objects used to resolve organism designations in
-            features. If empty, uses: :attr:`DEFAULT_ORGANISMS`.
-        :param types: A list or tuple of :class:`Type` objects used to resolve type designations in features.
-            If empty, uses: :attr:`DEFAULT_TYPES`.
-        :param kwargs: Any additional attributes, such as `fusion_strategy`, used with :class:`Genotype`.
-        :return: A :class:`Genotype` instance with the parsed gnomic genotype.
-        """
-        changes = cls._parse_string(string,
-                                    organisms or DEFAULT_ORGANISMS,
-                                    types or DEFAULT_TYPES)
-        return Genotype(changes, parent=parent, **kwargs)
-
-    @staticmethod
-    def chain_parse(definitions, **kwargs):
-        genotype = Genotype.parse(definitions[0], **kwargs)
-        for definition in definitions[1:]:
-            genotype = Genotype.parse(definition, parent=genotype, **kwargs)
-        return genotype
-
-    def _iter_changes(self, fusions=True):
-        if fusions:
-            for feature in self.added_fusion_features:
-                yield Ins(feature)
-
-            for feature in self.removed_fusion_features:
-                yield Del(feature)
-        else:
-            for feature in self.added_features:
-                yield Ins(feature)
-
-            for feature in self.removed_features:
-                yield Del(feature)
-
-        for plasmid in self.added_plasmids:
-            yield plasmid
-
-        for plasmid in self.removed_plasmids:
-            yield Del(plasmid)
-
-    # TODO some getter for accessing the original changes (self._changes)
-
-    def changes(self, fusions=False):
-        """
-        :param bool fusions: Keeps fusions together if ``True``, otherwise includes them as individual features.
-        :return:
-        """
-        return set(self._iter_changes(fusions=fusions))
-
-    def format(self, fusions=True, output='text'):
-        """
-        :param bool fusions: Keeps fusions together if ``True``, otherwise includes them as individual features.
-        :param str output: Output format; one of ``'text'``, ``'string'`` and ``'html'``
-        :return:
-        """
-        if output == 'text':
-            return genotype_to_text(self, fusions=fusions)
-        elif output == 'string':
-            return genotype_to_string(self, fusions=fusions)
-        elif output == 'html':
-            return genotype_to_html(self, fusions=fusions)
-        else:
-            raise NotImplementedError('Unknown output format: {}'.format(output))
-
-    def __repr__(self):
-        return '{}({})'.format(self.__class__.__name__, repr({'parent': self.parent, 'changes': self._changes}))
\ No newline at end of file
+from .genotype import Genotype  # noqa
```

Comment:
The import statement exists to make it easier to access Genotype. To signal that it is there to be exported and at the same time quiet flake8, specify it in `__all__`:

```
from .genotype import Genotype

__all__ = ('Genotype',)
```